### PR TITLE
little more reformating and add comments

### PR DESCRIPTION
**Pull request by IdanTo1**
last pull request for the project!
Signed-off-by: Idan Tomer <idan.tomer2@gmail.com>

### CODE REVIEW

File: src/PopularTag.cpp [hunk 1]
```diff
@@ -27,8 +27,8 @@ void PopularTag::increaseCount() {
 
 // overloaded < operator
 bool PopularTag::operator<(const PopularTag& pt) {
-    // check if pt is more popular, has bigger count (meaning User has more Watchables in is
-    //  hisorty containing this tag)
+    // check if pt is more popular, has bigger count (meaning User has more Watchables in his hisorty
```
Comment (Hroy02) [inline src/PopularTag.cpp]: History*, And maybe also it's instead of his because this is a user object and thus inanimate

**Review by Hroy02 — CHANGES_REQUESTED** [review]
A single point of grammar. Otherwise Perfect

File: src/PopularTag.cpp (hunk 1 above)
Comment (IdanTo1) [reply src/PopularTag.cpp]: actually it talks about the user as a person :sweat_smile: 
fixed the grammar problem

**Review by Hroy02 — APPROVED** [review]
Not very PC to assume the user is a male ;)

**Pull request merged**